README.md

```diff
@@ -24,31 +24,33 @@ Api group | @apiGroup
 Api permission | @apiPermission
 Api parameters | @apiParam
 On success response | @apiSuccess
-On error response | @apiError 
+On error response | @apiError
 
 
 
 ## CI
 
-This repository is configured for Continuous Integration (CI) with Travis CI. Click the above Build Status link for more details on the last build. 
+This repository is configured for Continuous Integration (CI) with Travis CI. Click the above Build Status link for more details on the last build.
 The CI is programmed to run all tests for the API, and deploy the current version of the API to EB, as well as deploys the frontend from website directory to Firebase.
 All deployments happen on the _master_ branch. Therefore, as much as possible, commits should not be made directly to _master_. All development should be run on _dev_
-and then PRed into _master_. 
+and then PRed into _master_.
 
 To run the tests, run `npm test` in the main directory.
+To compile and run the main website, run `npm run watch-sass` in the main directory.
+To run anything from app.hackpsu.org, run `npm run start` in the user-registration-app directory.
 
 ## Testing
 
-Testing is handled by the CI platform Travis-CI. Tests are Javascript files under api/test. Tests should use the [MochaJS](http://Mochajs.org) testing framework. In addition, use 
+Testing is handled by the CI platform Travis-CI. Tests are Javascript files under api/test. Tests should use the [MochaJS](http://Mochajs.org) testing framework. In addition, use
 [ChaiJS](http://chaijs.com) for assertions. RESTful testing is handled by the [chai-http](http://chaijs.com/plugins/chai-http/) module.
```

Comment:
RESTful testing also uses the [mocha-typescript](https://www.npmjs.com/package/mocha-typescript) package.
Formatted using \[mocha-typescript\](https://www.npmjs.com/package/mocha-typescript)

Reply:
Done